langchain/text_splitter.py

```diff
@@ -64,10 +64,15 @@ def create_documents(
                 documents.append(new_doc)
         return documents
 
-    def split_documents(self, documents: List[Document]) -> List[Document]:
+    # can be an iterable or a sequence
+    def split_documents(
+        self, documents: Union[Iterable[Document], Sequence[Document]]
```

Comment:
Is sequence an instance of iterable?

Reply:
Yes @eyurtsev you are right, will modify.